src/packages/configuration-service.ts

```diff
@@ -1,6 +1,9 @@
 import { getLogger } from "..";
 import config from "../config";
-import { Errors, identityUtils, nameservice } from "../packages";
+import {ErrorHelper, PackageErrorCode} from "./error";
+import * as identityUtils from "./identity-utils";
+import * as nameservice from "./nameservice";
+import {BlockstackService, UPLOADABLE_JSON_FILES} from "./nameservice";
```

Comment:
- [ ] always, can we import the package and use all the required methods/properties from the package reference?